run_clang_tidy.sh

```diff
@@ -6,6 +6,9 @@ exit_code=0
 FILES=$1
 
 for changed_file in ${FILES}; do
+  if [[ "${changed_file}" =~ /pdh/ || "${changed_file}" =~ /windows-event-log/ ]]; then
+    continue
+  fi
   if [[ "${changed_file}" == *.cpp ]]; then
     clang-tidy-14 -warnings-as-errors=* -quiet -p build "${changed_file}"
```

Comment:
I have added this as a quick-and dirty way of skipping the Windows-only extensions.  @lordgamez please let me know if you can think of a better way.